src/gcodeExport.cpp

```diff
@@ -1726,6 +1726,10 @@ void GCodeExport::finalize(const char* endCode)
     for (int n = 1; n < MAX_EXTRUDERS; n++)
         if (getTotalFilamentUsed(n) > 0)
             spdlog::info("Filament {}: {}", n + 1, int(getTotalFilamentUsed(n)));
+}
+
+void GCodeExport::writeGcodeFile()
```

Comment:
I'm not sure the name of this function is completely descriptive of what it does.

Reply:
I changed the name to flushOutputStream and added a bit  of description as well.